msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -233,6 +250,25 @@ Upon first run of local debug to Office or Outlook, you'll be prompted to sign i
 
 Provide feedback and report any issues with the Teams Toolkit debugging experience at [Microsoft Teams Framework (TeamsFx)](https://github.com/OfficeDev/TeamsFx/issues).
 
+#### Mobile debugging
+
+Here's how to remotely debug your app running in Office app for Android.
```

Comment:
Please change to:
To remotely debug your app running in Office app for Android.